nevergrad/functions/images/core.py

```diff
@@ -236,6 +239,7 @@ def _loss(self, x: np.ndarray) -> float:
         return loss
 
     def _generate_images(self, x: np.ndarray) -> np.ndarray:
+        """ generates images tensor of shape [nb_images, x, y, 3] with pixels between 0 and 255"""
         # pylint: disable=not-callable
         noise = torch.tensor(x.astype('float32'))
-        return self.pgan_model.test(noise).permute(0, 2, 3, 1).cpu().numpy()  # type: ignore
+        return ((self.pgan_model.test(noise).clamp(min=-1, max=1) + 1) * 255 / 2).permute(0, 2, 3, 1).cpu().numpy()  # type: ignore
```

Comment:
just to be sure, there is no constraint on `noise` / `x` ? You did not impose any